frontend_v2/src/app/components/nav/header-static/header-static.component.ts

```diff
@@ -88,7 +99,7 @@ export class HeaderStaticComponent implements OnInit, OnDestroy {
     private apiService: ApiService,
     @Inject(DOCUMENT) private document: Document
   ) {
-    this.authState = authService.authState;
```

Comment:
[Question] Why are we removing this line?

Reply:
This was removed by mistake. Sorry!